README.md

```diff
@@ -5,4 +5,11 @@ Simple Minecraft Server Install Script (Linux)
 - Get the script with `wget https://raw.githubusercontent.com/MrSlimeDiamond/minecraftsi/main/minecraftsi.sh`
 - Give yourself permission with `chmod +x minecraftsi.sh`
 - Run it with `./minecraftsi.sh`
-- You might have to install the `dialog` package to use.
+
+or
+
+Run script with `bash -c "$(curl -fsSL https://raw.githubusercontent.com/MrSlimeDiamond/minecraftsi/main/minecraftsi.sh)"`
+
+## Requirements
+
+- `dialog` - installable from apt and yum
```

Comment:
"Installable via your package manager"

Reply:
How do you mean it?

Reply:
To change it to something along those lines